Core/AVIDump.cpp

```diff
@@ -0,0 +1,360 @@
+// Copyright 2009 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#if defined(__FreeBSD__)
+#define __STDC_CONSTANT_MACROS 1
+#endif
+
+#include <string>
+#include <sstream>
+
+extern "C" {
+#include <libavcodec/avcodec.h>
+#include <libavformat/avformat.h>
+#include <libavutil/mathematics.h>
+#include <libswscale/swscale.h>
+}
+
+#include "Common/FileUtil.h"
+#include "Common/MsgHandler.h"
+#include "Common/ColorConv.h"
+
+#include "Core/Config.h"
+#include "Core/AVIDump.h"
+#include "Core/System.h"
+#include "Core/Screenshot.h"
+
+#include "GPU/Common/GPUDebugInterface.h"
+#ifdef _WIN32
+#include "GPU/Directx9/GPU_DX9.h"
```

Comment:
Seems like this include isn't used.  Might be cleaner to remove it and GLES?

-[Unknown]